pattern-1/marathon-applications/wso2apim-analytics.json

```diff
@@ -0,0 +1,100 @@
+{
+  "id": "wso2/apim-analytics",
+  "cmd": null,
+  "cpus": 1,
+  "mem": 2048,
+  "disk": 0,
+  "instances": 1,
+  "acceptedResourceRoles": [
+    "*"
+  ],
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "forcePullImage": false,
+      "image": "wso2am-analytics-dcos:2.1.0",
```

Comment:
Shall we use one common naming convention here ? saw that in some places, we use wso2apim-.... and here it's wso2am-..... ? WDYT ?

Reply:
We are using `wso2am-` in the docker-apim repo, I don't see an issue here. See

- https://github.com/wso2/docker-apim/tree/master/dockerfiles/apim
- https://github.com/wso2/docker-apim/tree/master/dockerfiles/apim-analytics

Reply:
Agree with Shariq! Since the product code name is still wso2am, Docker images have been named the same way.